src/test/test.cmd

```diff
@@ -13,6 +13,8 @@
 
 @call burn\test_burn.cmd %_C% %_T% || exit /b
 
+msbuild -t:Restore dtf\DtfE2ETests.sln -p:Configuration=%_C% -nologo -m -warnaserror -bl:%_L%\dtfe2etests.binlog || exit /b
```

Comment:
You just want to Restore, not build or test?

Reply:
Heh, no. I'll change that in my next PR.